REACT-APP/movies/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './App.css';
+import MoviesList from './components/MoviesList';
+import MovieInfo from './components/MovieInfo';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <div className="App">
+          <Route exact path="/" component={MoviesList}/>
+          <Route path="/:id" component={MovieInfo}/>
```

Comment:
Here, I would change the route path url to be a bit more descriptive of what the resource at this location is. So it should probably be something like `"/movies/:id"`.